.idea/codeStyles/Project.xml

```diff
@@ -1,12 +1,8 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <codeStyleSettings language="kotlin">
+    <JetCodeStyleSettings>
```

Comment:
Changes here WAI?

Reply:
Yep, this is the new structure for code styles in IntelliJ.